MTB_SCORE/score/compute_score.py

```diff
@@ -0,0 +1,84 @@
+# MTB Scoring
+
+# import the required packages
+import pandas as pd
+from MTB_SCORE.util import util as ut
+from MTB_SCORE.util.get_Stepdata import Get_Stepdata
+from MTB_SCORE.util.get_score_item import Get_score_items
+from MTB_SCORE.util.get_Scoresums import Get_Scoresum
+from MTB_SCORE.score.get_scores import Get_Score
+from MTB_SCORE.score.get_spellvocab_score import GetScore_SpellingAndVocab
+
+
+def get_drop_dict(t_type):
+    config = ut.get_config()
```

Comment:
This function has to purposes. Config file does three things - mapping of terms, constants, mapping of values for certain measures.

This fucntion is mixing the config work with the drop list.  Suggest splitting into two.  get_config and get_drop_dict.

Add a doc function to explain.